Source/WebKitLegacy/mac/WebView/WebHTMLView.mm

```diff
@@ -7141,6 +7147,22 @@ - (BOOL)_findString:(NSString *)string options:(WebFindOptions)options
     return coreFrame && coreFrame->editor().findString(string, coreOptions(options));
 }
 
+#if ENABLE(WRITING_TOOLS)
+
+// Disable Writing Tools in WebKitLegacy.
+
+- (PlatformWritingToolsBehavior)writingToolsBehavior
+{
+    return PlatformWritingToolsBehaviorNone;
+}
+
+- (BOOL)providesWritingToolsContextMenu
```

Comment:
This could use some clarification (with a comment), because we're saying "no Writing Tools" but the code appears to be saying "I provide a Writing Tools menu item" 😅